GPUSimulators/Common.py

```diff
@@ -548,7 +556,7 @@ def download(self, stream, cpu_data=None, async=False, extent=None):
         copy.height = int(ny)
         
         copy(stream)
-        if async==False:
+        if asynch==False:
```

Comment:
As above - not commenting all other references to async

Reply:
This change is necessary for newer versions of Python, see inducer/pycuda#166